monero-outreach-docs/translations/fr/archive/rpc_pay_fr.md

```diff
@@ -0,0 +1,133 @@
+
+### _Regard Vers L'avenir_
+
+Le Paiement-RPC a été intégré pour la première fois à la version 0.15 de Monero. Il est tout neuf. Avec le temps, des nouvelles fonctionnalités seront ajoutées : les applications potentielles du Paiement-RPC évolueront avec le système lui-même. Primo est la première application externe, qui servira d'exemple aux autres. Tout serveur sur l'Internet qui fournit des données ou des calculs aux clients est un client potentiel pour l'utilisation du Paiement-RPC à des fins de financement. Avec RandomX comme algorithme de preuve de travail de Monero, aussi inclus pour la première fois dans la version 0.16, tout ordinateur peut calculer des hash de minage pour alimenter le Paiement-RPC. Le temps est venu pour cette nouvelle fonctionnalité, et elle entrainera plusieurs développements excitants.
+
+### _Guides de Démarrage_
+
+##### _#1: Exemple de Paiement-RPC Monero_
+---
+
+En utilisant le curl de la ligne de commande ([curl.haxx.se](https://curl.haxx.se/)), vous pouvez envoyer des appels RPC Monero à votre propre instance monerod ou à un serveur publique. Par exemple, si vous voulez savoir combien de blocs composent la chaîne de blocs Monero, vous pouvez utiliser la commande RPC get_block_count. Pour obtenir cette information à partir d'un serveur publique chez Monero World, installez curl et entrez la commande suivante à la ligne de commande:
+
+```
+curl -X POST [uwillrunanodesoon.moneroworld.com:18089/json_rpc](http://uwillrunanodesoon.moneroworld.com:18089/json_rpc) -H 'Content-Type:application/json' --data "{\"jsonrpc\":\"2.0\",\"id\":\"0\",\"method\":\"get_block_count\"}"
+```
+
+Vous aurez une réponse qui ressemble à ceci:
+
+```
+{ "id": "0", "jsonrpc": "2.0", "result": { "count": 2021560, "status": "OK", "untrusted": false } }
+```
+
+Le nombre de blocs dans la chaine de blocs était de 2,021,560 lorsque cette sortie a été générée.
+
+Si vous faites rouler votre propre instance de _monerod_ en configuration standard, vous pouvez obtenir la même information avec les commandes suivantes (similaires) sur le même ordinateur:
+
+```
+curl -X POST [127.0.0.1:18081/json_rpc](http://127.0.0.1:18081/json_rpc) -H 'Content-Type:application/json' --data "{\"jsonrpc\":\"2.0\",\"id\":\"0\",\"method\":\"get_block_count\"}"
+```
+
+Pour plus d'information à propos de l'interface entre _monerod_ et RPC, voir [www.getmonero.org/resources/developer-guides/daemon-rpc.html](https://www.getmonero.org/resources/developer-guides/daemon-rpc.html). Pour plus d'information à propos de l'interface entre le portefeuille et RPC, voir [www.getmonero.org/resources/developer-guides/wallet-rpc.html](https://www.getmonero.org/resources/developer-guides/wallet-rpc.html).
```

Comment:
Here change the link to a French getmonero link: https://www.getmonero.org/fr/resources/developer-guides/wallet-rpc.html
[www.getmonero.org/fr/resources/developer-guides/wallet-rpc.html](https://www.getmonero.org/fr/resources/developer-guides/wallet-rpc.html)

Reply:
Integrated